images/metering-ansible-operator/roles/meteringconfig/tasks/validate_reporting_operator_tls.yml

```diff
@@ -23,4 +23,3 @@
         - meteringconfig_spec['reporting-operator'].spec.authProxy.authenticatedEmails.data != ""
       msg: "spec.reporting-operator.spec.authProxy.authenticatedEmails: secretName and/or data key(s) cannot be empty when enabled and createSecret is set to true"
     when: meteringconfig_spec['reporting-operator'].spec.authProxy.authenticatedEmails.enabled and meteringconfig_spec['reporting-operator'].spec.authProxy.authenticatedEmails.createSecret
-  when: not meteringconfig_tls_enabled
```

Comment:
Won't this break when TLS is disabled then?

Reply:
Nope - this at was redundant:
```
- name: Validate the user-provided authProxy configuration
  include_tasks: validate_reporting_operator_tls.yml
  when: not meteringconfig_tls_enabled
```

Reply:
Nope - that was redundant:

- name: Validate the user-provided authProxy configuration
  include_tasks: validate_reporting_operator_tls.yml
  when: not meteringconfig_tls_enabled

Reply:
Ah